samples/native-lib/README.md

```diff
@@ -49,14 +49,14 @@ will be generated.
 
 ```bash
 cd build
-./iwasm --native-lib=libtest_add.so --native-lib=libtest_sqrt.so wasm-app/test.wasm
+./iwasm --native-lib=./libtest_add.so --native-lib=./libtest_sqrt.so --native-lib=./libtest_hello.so wasm-app/test.wasm
 ```
 
 ### macOS
 
 ```bash
 cd build
-./iwasm --native-lib=libtest_add.dylib --native-lib=libtest_sqrt.dylib wasm-app/test.wasm
+./iwasm --native-lib=libtest_add.dylib --native-lib=libtest_sqrt.dylib --native-lib=libtest_hello.dylib wasm-app/test.wasm
```

Comment:
Should also modify the output below, add:
```
test_hello("main", (nil), 0) = 41
malloc(42) = 0x24b8
test_hello("main", 0x24b8, 42) = 41
```

Reply:
sure. done.